contract_compassion/view/contract_view.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data>
+		<record id="view_recurring_contract_tree_compassion" model="ir.ui.view">
+			<field name="name">recurring.contract.compassion.tree</field>
+			<field name="model">recurring.contract</field>
+			<field name="inherit_id" ref="recurring_contract.view_recurring_contract_tree"/>
+			<field eval="18" name="priority"/>
+			<field name="arch" type="xml">
+				<data>
+					<tree string="Contracts" position="attributes">
+						<attribute name="colors">blue:state == 'draft';red:state == 'mandate';green:state == 'waiting';black:state == 'active';gray:state in ('terminated','cancelled')</attribute>
+						<attribute name="string">Sponsorships</attribute>

```

Comment:
Remove this override. See how it can be made in sponsorship_compassion. We should have two different menus for contracts and sponsorships.